src/plfa/part1/Relations.lagda.md

```diff
@@ -649,7 +649,7 @@ the fact that inequality is transitive.
 ```
 
 
-## Even and odd
+## Even and odd {#even-and-odd}
```

Comment:
This shouldn't be necessary. That anchor is automatically generated.

Reply:
Ok. I guess the generated name will be `#Even-and-odd`?

Reply:
I think it removes all capitalization, but it should be easy to check on the live webpage.

Reply:
True! I just checked, it is `#even-and-odd`. 😃